modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/StepsControlButtons.scala

```diff
@@ -29,6 +29,127 @@ import seqexec.web.client.semanticui.elements.icon.Icon.IconTrash
 import seqexec.web.client.reusability._
 import web.client.ReactProps
 
+/**
+  * Contains a set of control buttons like stop/abort
+  */
+final case class ControlButtons(
+  id:              Observation.Id,
+  operations:      List[Operations[_]],
+  sequenceState:   SequenceState,
+  stepId:          Int,
+  isObservePaused: Boolean,
+  tabOperations:   TabOperations
+) extends ReactProps {
+  @inline def render: VdomElement = ControlButtons.component(this)
+
+  val requestInFlight = tabOperations.stepRequestInFlight
+}
+
+object ControlButtons {
+  type Props = ControlButtons
+
+  implicit val operationsReuse: Reusability[Operations[_]] = Reusability.derive[Operations[_]]
```

Comment:
I was bit surprised this'd work but then noted that `Operations[T]` doesn't have an internal reference to T. Is that why this compiles?

Reply:
TBH, I was a bit surprised too. Probably that's the reason, since a `Reusability[T]` is not required.